ui/src/session.ts

```diff
@@ -176,14 +158,14 @@ export const addSeed = async (seed: string): Promise<boolean> => {
   if (get(seedValidation).status !== ValidationStatus.Success) return false;
 
   updateCoCo({ seeds: [...get(settings).coco.seeds, seed] });
-  showRestartNotification();
+  notification.info("New Seed added");
```

Comment:
I don't think we need notifications about this. Just adds noise, the user can already see in the UI that it's added or removed from the list.